boot/freeldr/freeldr/arch/i386/pcmem.c

```diff
@@ -331,7 +331,7 @@ PcMemGetBiosMemoryMap(PFREELDR_MEMORY_DESCRIPTOR MemoryMap, ULONG MaxMemoryMapSi
         TRACE("BaseAddress: 0x%llx\n", PcBiosMemoryMap[PcBiosMapCount].BaseAddress);
         TRACE("Length: 0x%llx\n", PcBiosMemoryMap[PcBiosMapCount].Length);
         TRACE("Type: 0x%lx\n", PcBiosMemoryMap[PcBiosMapCount].Type);
-        TRACE("Reserved: 0x%lx\n", PcBiosMemoryMap[PcBiosMapCount].Reserved);
+        TRACE("EA: 0x%lx\n", PcBiosMemoryMap[PcBiosMapCount].EA);
```

Comment:
This is most likely undefined behavior. AsULONG will help you here as well.

Reply:
Done.